_drafts/Introducing-geoextent.md

```diff
@@ -0,0 +1,131 @@
+# Introducing geoextent
+
+geoextent is an easy to use library for extracting the geospatial extent from data files with multiple data formats.
+
+Take a look at the [source code on GitHub](https://github.com/o2r-project/geoextent.git), [library on PyPI](https://pypi.org/project/geoextent/) and the [documentation website](https://o2r.info/geoextent/).
+
+Here is a small example how to use `geoextent`.
+```
+geoextent -b -t -input= 'cities_NL.csv'
```

Comment:
Add a link where a reader can view the used input file, please.

Reply:
Done.